huffman.go

```diff
@@ -203,21 +81,18 @@ func (huff *Huffman) Decompress(data []byte) ([]byte, error) {
 			break
 		}
 
-		dst = append(dst, n.symbol)
+		dst = append(dst, n.Symbol)
 	}
 
 	return dst, nil
 }
 
 func (huff *Huffman) Compress(data []byte) ([]byte, error) {
-	if huff.initalized == false {
-		huff.Init()
-	}
 
 	srcIndex := 0
 	end := len(data)
 	bits := uint32(0)
-	bitcount := uint32(0)
```

Comment:
Ou nice does this fix a bug?

Reply:
no it's one just doesn't need 32 bits for that type.